src/Queue/Queue.php

```diff
@@ -91,6 +79,7 @@ public function createItem($data) {
   public function numberOfItems() {
     // Retrieve information about the queue without modifying it.
     $queue_options = ['passive' => TRUE];
+    unset($this->queue);
```

Comment:
You may set this class defined property to null and let GC handle object deletion